starter-code/styles/icons.css

```diff
@@ -68,3 +68,9 @@
 .icon-github::before {
 	content: "\e907";
 }
+
+#hamburger {
+	width: 5%;
+	height: 5%;
+	float: right;
```

Comment:
Great work floating this to the right (although, as per my comments above, this should be applied to the .icon-menu class instead of your hamburger :))!